addons/account/models/account_move.py

```diff
@@ -2280,11 +2280,34 @@ def is_internal_partner(partner):
         move.message_subscribe(list(all_followers_ids))
         return move
 
-    def post(self):
+    def post(self, soft=True):
```

Comment:
```suggestion
    def _post(self, soft=True):
          ............
    def post(self):
        logger.log('deprecated method')
        return self._post()

```
so that we force people to consider the new parameter (or use action_post)

Reply:
I understand why we would do that in stable, but then api changes all the time in master and we never do things like that; why would we do it here?

Reply:
well, you're right for the depreciation warning... but the privatization of the method is right, given that the exposed method to call is supposedly action_post(). :thinking: 

fifty-fifty, and we just replace

``` diff
-    def post(self, soft=True):
+    def _post(self, soft=True):
```
?

Reply:
yeah, :smile: 

if you want a tip about it: I would personally highlight the remaining occurrences by doing
``` python
    def _post(self, soft=True):
          ............
    def post(self):
        logger.log('deprecated method')
        return self._post()
```

ho... wait .. :grin: